esp_hosted_ng/host/esp_sdio.dts

```diff
@@ -0,0 +1,55 @@
+/dts-v1/;
+/plugin/;
+
+/* Enable SDIO from MMC interface via various GPIO groups */
```

Comment:
Every device will have their own way of defining the device tree.
Although, We can add this part of device tree as example device tree, please help add as it is seen as example. for example, add the sub-directory example/<specific_device_name> and move there.

Same is the case for spi dts.

Reply:
May be in this case, `example/raspberry_pi_3b+` etc

Reply:
> Every device will have their own way of defining the device tree. 

Yes, I understand this is essentially a copy of boot/overlay/sdio-overlay.dtso with the parameters from the docs.
And some added configs for default options.
Although, We can add this part of device tree as example device tree, please help add as it is seen as example. for example, add the sub-directory example/<specific_device_name> and move there.
> 
> Same is the case for spi dts.

Reply:
it will be definitely usable for users when they use device tree as their mode of configuration (For Raspberry Pi, it is not default for most users). It needs to be restructured to showcase as example as stated above.